### PR TITLE
docs(pt): Create Portuguese VitePress translation

### PR DESCRIPTION
**Pull request by viniciusdeliz**
- [x] ~Translate `guide/markdown.md`~
- [x] ~Translate `reference/frontmatter-config.md`~
- [x] ~Translate `reference/cli.md`~
- [x] ~Translate `reference/default-theme-config.md`~
- [x] ~Translate `reference/default-theme-nav.md`~
- [x] ~Translate `reference/default-theme-sidebar.md`~
- [x] ~Translate `reference/default-theme-homepage.md`~
- [x] ~Translate `reference/default-theme-layout.md`~
- [x] ~Translate `reference/default-theme-badge.md`~
- [x] ~Translate `reference/default-theme-team-page.md`~
- [x] ~Translate `reference/default-theme-prev-next-links.md`~
- [x] ~Translate `reference/default-theme-edit-link.md`~
- [x] ~Translate `reference/default-theme-last-updated.md`~
- [x] ~Translate `reference/default-theme-footer.md`~
- [x] ~Translate `reference/default-theme-search.md`~
- [x] ~Translate `reference/default-theme-carbon-ads.md`~

### CODE REVIEW

**Comment by viniciusdeliz** [comment]
@savionascimentodev your help would be of great value in here! There's a lot of English content to be translated

**Comment by savionascimentodev** [comment]
You can count on me 😁

**Comment by viniciusdeliz** [comment]
> You can count on me 😁

@savionascimentodev You can pick an item here from the list above and let me know here so I can update its status. If you prefer we can use the Issues inside my fork https://github.com/viniciusdeliz/vitepress/issues

**Comment by viniciusdeliz** [comment]
Mentioning @brc-dd because I saw he reviewed many of the docs translation/i18n PRs.
I'd love to know if this PR is adequate  for including a Portuguese version of the VitePress docs. I'll be glad to collaborate.
Let me know if I should perform any adjustment, or if I should include any file or any change at all. Thanks!

File: docs/pt/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+titleTemplate: Gerador de Site Estático desenvolvido com Vite & Vue
+
+hero:
+  name: VitePress
+  text: Gerador de Site Estático Vite & Vue
+  tagline: Simples, poderoso e rápido. Conheça o moderno framework SSG que você sempre quis.
+  actions:
+    - theme: brand
+      text: Iniciar
+      link: /guide/getting-started
```
Comment (awxiaoxian2020) [inline docs/pt/index.md:14]:
```suggestion
      link: /pt/guide/getting-started
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

**Comment by brc-dd** [comment]
Please update docs/lunaria.config.json too.

**Comment by viniciusdeliz** [comment]
> Please update docs/lunaria.config.json too.

@brc-dd updated and running. Let me know if further adjustments are required!

**Comment by brc-dd** [comment]
Hi, the ids of heading should be synced to the English version. Refer Chinese counterparts for example.

**Comment by viniciusdeliz** [comment]
> Hi, the ids of heading should be synced to the English version. Refer Chinese counterparts for example.

@brc-dd I've just included the heading links in all the files. And thank you for the addition that I missed in the `lunaria.config.json`. Let me know if I missed anything else, I'll be glad to assist

**Comment by brc-dd** [comment]
Thanks. I'll review this in few days.

**Pull request merged**